example/gluon/word_language_model/README.md

```diff
@@ -1,67 +1,59 @@
 # Word-level language modeling RNN
 
-This example trains a multi-layer RNN (Elman, GRU, or LSTM) on Penn Treebank (PTB) language modeling benchmark.
+This example trains a multi-layer RNN (Elman, GRU, or LSTM) on WikiText-2 language modeling benchmark.
 
-The model obtains the state-of-the-art result on PTB using LSTM, getting a test perplexity of ~72.
-And ~97 ppl in WikiText-2, outperform than basic LSTM(99.3) and reach Variational LSTM(96.3).
+The model obtains ~97 ppl in WikiText-2 using LSTM, outperforming basic LSTM(99.3) and reach Variational LSTM(96.3).
```

Comment:
The current default setting is problematic. After running the experiment, the best testing ppl is 106.9

Reply:
Will update the number. Let's find a decent setting and put it back in later.